src/assets/components/Nav/Nav.jsx

```diff
@@ -0,0 +1,17 @@
+import { Link } from "react-router-dom";
+import "./Nav.css";
+function Nav() {
+  return (
+    <nav className="navigation">
+      <div>
+        <h1>Book Swap</h1>
+      </div>
+      <div className="links">
+        <Link to="availableBooks">Available books</Link>
+        <Link to="claimedBooks">Claimed books</Link>
+        <Link to="addBook">Add book</Link>
```

Comment:
I think we'll want these 'Link' items to be 'NavLinks', as it adds the .active class to the element for CSS targeting, so we will be able to highlight which section we are on, e.g.:

```
// The Navlink component is the same as a link, but it automatically 
// applies a class of "active" to whichever one is active 
<nav> 
<NavLink to="/">Home</NavLink> 
<NavLink to="about">About</NavLink> 
<NavLink to="blog">Blog</NavLink> 
</nav> 
```

Reply:
Looks good - we may have to update once we have looked at the subsequent stories, to see how the data filtering works for available / claimed books, but think we will do that when we get to those stories.